### DISCUSSION TITLE
Not syncing

### DISCUSSION THREAD

**Discussion by ghost** [Chia-Network/chia-blockchain#9044] [Support]
My house had a power outage in the early morning hours yesterday and I have not been able to get synced since. I just get the spinning circles of doom forever. I've tried un-installing/re-installing chia app. I tried different versions of the app. I the completely deleted all chia from computer and installed again. That seemed to work but had to download full blockchain. It started downloading and was going well, however when it was about 75% downloaded the computer rebooted and haven't been able to get past spinning circles since... any suggestions would be much appreciated

**Comment by Tomaszu97** [comment]
I had sync issues for over a month since I've tried to do it on an rpi 4, it was just way too slow. Use a quicker hardware (preferably x86 architecture) to run full node.
Check your log file to see if anything's happening (your home directory -> .chia/mainnet/logs/debug.log and also debug.log.1, .2, .3 etc...
You should see some activity regarding received blocks from other peers.

You can increase log verbosity by using `chia configure --set-log-level DEBUG` using... cli :) otherwise, what's written there by default might not be enough. I heavily recommend using chia cli instead of gui - this way you have more control over what's happening instead of just crossing your fingers for success and looking at super slow electron app with a spinning circle.

**Comment by BrandtH22** [comment]
If you are still experiencing the issues it is recommended to upgrade to the latest version of chia. Please download it from here: https://www.chia.net/downloads/

Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
